lib/aot/compiler.cpp

```diff
@@ -40,6 +40,10 @@
 #include <llvm/Object/COFF.h>
 #endif
 
+#if LLVM_VERSION_MAJOR >= 15
```

Comment:
No needs to check version here, the header was presented before llvm 9

Reply:
@ibmibmibm 
We can use your PR instead #1863

Reply:
Oops, looks like you already deleted the branch, let me update this now.